demisto_sdk/commands/validate/tests/IN_validators_test.py

```diff
@@ -184,3 +238,1639 @@ def test_IsIntegrationRunnableValidator_is_valid(
+                    ],
+                ),
+                create_integration_object(
+                    paths=["configuration"],
+                    values=[
+                        [
+                            {
+                                "name": "proxy",
+                                "display": "Use system proxy settings",
+                                "type": 8,
+                                "required": True,
+                            }
+                        ]
+                    ],
+                ),
+            ],
+            4,
+            [
+                "The following params are invalid:\nThe proxy param display name should be 'Use system proxy settings', the 'defaultvalue' field should be 'False', the 'required' field should be 'False', and the 'required' field should be 8.",
+                "The following params are invalid:\nThe proxy param display name should be 'Use system proxy settings', the 'defaultvalue' field should be 'False', the 'required' field should be 'False', and the 'required' field should be 8.",
+                "The following params are invalid:\nThe proxy param display name should be 'Use system proxy settings', the 'defaultvalue' field should be 'False', the 'required' field should be 'False', and the 'required' field should be 8.",
+                "The following params are invalid:\nThe proxy param display name should be 'Use system proxy settings', the 'defaultvalue' field should be 'False', the 'required' field should be 'False', and the 'required' field should be 8.",
+            ],
+        ),
+    ],
+)
+def test_IsValidProxyAndInsecureValidator_is_valid(
+    content_items, expected_number_of_failures, expected_msgs
+):
+    """
+    Given
+    content_items iterables.
+        - Case 1: Four valid integrations:
+            - One integration without proxy / insecure params.
+            - One integration with valid insecure param.
+            - One integration with valid unsecure param.
+            - One integration with valid proxy param.
+        - Case 2: Four invalid integrations:
+            - One integration with proxy param with a wrong display name.
+            - One integration with proxy param without required field and a wrong type field.
+            - One integration with proxy param with a wrong type.
+            - One integration with proxy param with a required field set to true.
+    When
+    - Calling the IsValidProxyAndInsecureValidator is valid function.
+    Then
+        - Make sure the validation fail when it needs to and the right error message is returned.
+        - Case 1: Should pass all.
+        - Case 2: Should fail all.
+    """
+    results = IsValidProxyAndInsecureValidator().is_valid(content_items)
+    assert len(results) == expected_number_of_failures
+    assert all(
+        [
+            result.message == expected_msg
+            for result, expected_msg in zip(results, expected_msgs)
+        ]
+    )
```

Comment:
IMO its better to do not as list comprehension, its harder to debug it if needed in the future. That's true for any test, not only for this specific one.